lightwood/api/data_source.py

```diff
@@ -345,35 +353,25 @@ def _prepare_encoders(self):
         with all available data for that column.
         """
         encoders = {}
-    
         input_encoder_training_data = {'targets': [], 'previous': []}
 
-        for config in self.config['output_features']:
```

Comment:
It's not deleted, I just changed the order (see line 394 in the incoming changes). I had to opt for this because we need to setup the group normalizers before preparing output encoders (if they turn out to be `TsNumericEncoders`, that info needs to exist first).